tests/test_notebook.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
The tests should be extended to be much more extensive and cover more of the `notebook.py` file. Just testing initialization is not sufficient.